pkg/webhook/sidecarset/validating/sidecarset_create_update_handler.go

```diff
@@ -318,6 +351,21 @@ func validateSidecarConflict(sidecarSets *appsv1alpha1.SidecarSetList, sidecarSe
 		}
 	}
 
+	// whether pod metadata conflict
+	for _, patch := range sidecarSet.Spec.PatchPodMetadata {
+		patchType := patch.PatchPolicy
+		for key := range patch.Annotations {
+			if other, ok := annotationsInOthers[key]; ok {
+				slice := strings.Split(other, "#")
+				if patchType == appsv1alpha1.OverwritePatchPolicy || patchType == appsv1alpha1.RetainPatchPolicy ||
```

Comment:
Overwrite is conflicting with Overwrite, but if both sidecarset use Retain policy, it have no conflicts